.azure/pipelines/ci.yml

```diff
@@ -674,9 +674,9 @@ stages:
       # Build the shared framework
       - script: ./build.cmd -ci -all -pack -arch x64 -buildNative /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log /bl:artifacts/log/helix.build.x64.binlog
         displayName: Build shared fx
-      - script: .\restore.cmd -ci
+      - script: .\restore.cmd -ci /p:BuildInteropProjects=true
         displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:RunTemplateTests=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildInteropProjects=true /p:BuildNative=true /p:RunTemplateTests=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
```

Comment:
You will need to add this to all the various helix legs that you want these tests to run on (daily helix, and maybe the quarantined yml too since i imagine its possible some of these tests would get social distanced eventually too)